code/modules/religion/sects/shadow_sect.dm

```diff
@@ -301,3 +384,40 @@
 	organ.Insert(rite_target)
 	rite_target.visible_message("<span class='notice'>[organ] have been merged into [rite_target] by the rite of [name]!</span>")
 	return TRUE
+
+/datum/religion_rites/final_darkness
+	name = "Final Darkness"
+	desc = "The endgame, Activates the shadow heal and (if you meet the requirements) summons numerous Faithsworn entities from every obelisk in existence. THIS IS ONE USE ONLY."
+	ritual_length = 60 SECONDS
+	ritual_invocations = list(
+		"Join us in this darkness ...",
+		"... Protect us from the light ...",
+		"... Destroy those who defy us ...")
+	invoke_msg = "... Let the darkness come and fight!!"
+	favor_cost = 50000
+
+
+/datum/religion_rites/final_darkness/invoke_effect(mob/living/user, atom/religious_tool)
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	if(sect.faithful_used)
+		to_chat(user,"<span class='warning'>This rite has already been used, your favor has been refuned.</span>")
+		GLOB.religious_sect?.adjust_favor(50000, user)
+		return..()
+	sect.faithful_used = TRUE
+	for(var/obj/structure/destructible/religion/shadow_obelisk/obs in sect.obelisks)
+		obs.LateInitialize()
+	if(user.mind.is_murderbone())
+		priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+		sleep(150)
```

Comment:
ideally, instead of sleeping this would add a timer instead

Reply:
Makes sense, I'll sort it in the next 25 min

Reply:
I *think* I've sorted it